kyaml/yaml/merge2/merge2.go

```diff
@@ -39,7 +45,7 @@ type Merger struct {
 
 var _ walk.Visitor = Merger{}
 
-func (m Merger) VisitMap(nodes walk.Sources) (*yaml.RNode, error) {
+func (m Merger) VisitMap(nodes walk.Sources, s *openapi.ResourceSchema) (*yaml.RNode, error) {
```

Comment:
It appears that it was unnecessary to change the signatures of the `Visit*` methods in `merge2` and `merge3`, since the schema is associated with the Walker struct.

Reply:
yeah, how about we fix it in a follow up?  it is basically a no-op change, but is going to generate a big diff